fdbserver/MoveKeys.actor.cpp

```diff
@@ -3376,7 +3421,7 @@ ACTOR Future<Void> prepareBlobRestore(Database occ,
 		tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
 		tr.setOption(FDBTransactionOptions::LOCK_AWARE);
 		try {
-			wait(checkPersistentMoveKeysLock(&tr, lock));
```

Comment:
What's the difference between `checkPersistentMoveKeysLock` and `checkPersistentMoveKeysLockWhenPrepareBlobRestore`? Can we use a parameter to indicate the difference and we don't need to create a new function?

Reply:
The difference is the input lock. checkPersistentMoveKeysLock is a copy while checkPersistentMoveKeysLockWhenPrepareBlobRestore is a raw pointer.